### PR TITLE
Upgrade gradle v3.2.1

### PR DESCRIPTION
**Pull request by muga**
follow up of https://github.com/embulk/embulk/pull/464. 

Release notes of gradle v3.2.1:
https://docs.gradle.org/3.2.1/release-notes

### CODE REVIEW

**Comment by muga** [comment]
When I compile my plugin by gradle v3.2.1, the gradle shows the following warning. 
```
$ ./gradlew task
The Task.leftShift(Closure) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use Task.doLast(Action) instead.
        at build_rz67qtkuaofpxzi9co6hqne5.run(/path/to/my_plugin/build.gradle:67)
```

File: lib/embulk/command/embulk_migrate_plugin.rb [hunk 1]
```diff
@@ -43,8 +43,8 @@ def self.migrate_java_plugin(migrator, from_ver)
     end
 
     # upgrade gradle version
-    if migrator.match("gradle/wrapper/gradle-wrapper.properties", /gradle-2\.\d-/)
-      # gradle < 2.10 (\d matches one digit)
+    if migrator.match("gradle/wrapper/gradle-wrapper.properties", /gradle-2\.\d+-/)
```
Comment (hiroyuki-sato) [inline lib/embulk/command/embulk_migrate_plugin.rb]: 👍 Note.  
Next time, we will need to care the gradle version 2.X and < 3.2.1. 
ie) `/gradle-[23]\.\d+(\.\d+)?-/` (not tested yet).

Comment (muga) [inline lib/embulk/command/embulk_migrate_plugin.rb]: Thank you. Fixed 2fdd60670992e8ab3c32433912d51dbc8dcb60d0

Comment (muga) [inline lib/embulk/command/embulk_migrate_plugin.rb]: Yea, you're right. We will need it. (I didn't notice what you were saying before. Sorry about it.)

**Pull request merged**